### PR TITLE
Use ellipsis - part 1

### PR DESCRIPTION
**Pull request by brianteeman**
Following the style guide and accessibility recommendations this PR changes the use of ... (dot dot dot) to the html entity for an ellipsis.

For example some screenreaders will announce the three dots eg read more dot dot dot but with an ellipsis they pause and do not say dot dot dot

This PR is only for the language strings. A further PR will follow to address truncation

PARTIAL Pull Request for Issue #38630

### CODE REVIEW

File: administrator/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -643,7 +643,7 @@ JGLOBAL_SHOW_PARENT_CATEGORY_DESC="If set to Show, the title of the article&rsqu
 JGLOBAL_SHOW_PARENT_CATEGORY_LABEL="Parent Category"
 JGLOBAL_SHOW_PUBLISH_DATE_DESC="If set to Show, the date and time an Article was published will be displayed."
 JGLOBAL_SHOW_PUBLISH_DATE_LABEL="Publish Date"
-JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more ...Link will show if Main text has been provided for the Article."
+JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more &hellip;Link will show if Main text has been provided for the Article."
```
Comment (tecpromotion) [inline administrator/language/en-GB/joomla.ini]:
```suggestion
JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more &hellip; link will show if Main text has been provided for the Article."
```

File: api/language/en-GB/joomla.ini [hunk 2]
```diff
@@ -637,7 +637,7 @@ JGLOBAL_SHOW_PARENT_CATEGORY_DESC="If set to Show, the title of the article&rsqu
 JGLOBAL_SHOW_PARENT_CATEGORY_LABEL="Parent Category"
 JGLOBAL_SHOW_PUBLISH_DATE_DESC="If set to Show, the date and time an Article was published will be displayed."
 JGLOBAL_SHOW_PUBLISH_DATE_LABEL="Publish Date"
-JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more ...Link will show if Main text has been provided for the Article."
+JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more &hellip;Link will show if Main text has been provided for the Article."
```
Comment (tecpromotion) [inline api/language/en-GB/joomla.ini]:
```suggestion
JGLOBAL_SHOW_READMORE_DESC="If set to Show, the Read more &hellip; link will show if Main text has been provided for the Article."
```

**Review by tecpromotion — CHANGES_REQUESTED** [review]
Maybe this should be separated. Already hyphenated and lowercase elsewhere.

File: administrator/language/en-GB/joomla.ini (hunk 1 above)
Comment (brianteeman) [reply administrator/language/en-GB/joomla.ini]: good spot they should never have been capitalised

**Comment by brianteeman** [comment]
> Maybe this should be separated. Already hyphenated and lowercase elsewhere.

Sorry I dont understand what you are saying - maybe I need more coffee

**Review by tecpromotion — APPROVED** [review]
Thanks @brianteeman

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 56ac93c9a7f00fd5d2e6e20fb4e6c5d734ca0dae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38638">issues.joomla.org/tracker/joomla-cms/38638</a>.</sub>

**Comment by N6REJ** [comment]
@brianteeman docs will need to be updated as it specifically states "three periods"

**Comment by brianteeman** [comment]
Where? Search didn't find any. https://docs.joomla.org/index.php?title=Special:Search&profile=default&search=%22three%20periods%22&fulltext=1

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 56ac93c9a7f00fd5d2e6e20fb4e6c5d734ca0dae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38638">issues.joomla.org/tracker/joomla-cms/38638</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38638">issues.joomla.org/tracker/joomla-cms/38638</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38638">issues.joomla.org/tracker/joomla-cms/38638</a>.</sub>

**Comment by N6REJ** [comment]
@brianteeman https://developer.joomla.org/en-gb-user-interface-text-guidelines/punctuation.html
```
An ellipsis is a set of three periods ( ... ) indicating an omission. The group of periods should have a single space on either side. Unless it is at the end of a string where there is no space or period after.
Example

Read more ...
Create some text ... add some pictures.
```
Notice it says "three periods"

**Comment by brianteeman** [comment]
@N6REJ thanks - the docs have been updated just waiting for a merge

**Comment by roland-d** [comment]
Thank you

**Comment by brianteeman** [comment]
thanks

**Comment by Razzo1987** [comment]
Also in [COM_MEDIA_SEARCH ](https://github.com/joomla/joomla-cms/blob/835d9860c55a84b0275927b520ac0f604e4b92d1/language/en-GB/com_media.ini#L101)

**Comment by brianteeman** [comment]
@Razzo1987 thanks - see #38710

**Comment by HLeithner** [comment]
Whats the reason why you use the html encoding & hellip ; instead of the utf8 character … ?

**Comment by brianteeman** [comment]
Clarity.

**Comment by brianteeman** [comment]
No I will not change it and we have plenty of html in our strings. 

&hellip is easier to type and visually an editor will not mistakenly type ... when they mean …

As already explained we must use an ellipsis for accessibility and changing to use the utf8 … from the ... will not help as translators etc will not SEE that they are different and will continue to type ... especially as its much easier to type ... than it is to look up the utf8 character. It's certainly easier for me to type &hellip than it is to open a character map and locate the utf8 map and then locate the ellipsis and insert it into a document

**Comment by brianteeman** [comment]
image

which of those is the real ellipsis

**Comment by HLeithner** [comment]
That one which also works outside of a webbrowser.

**Comment by HLeithner** [comment]
OK so the Unicode consortium are all idiots and we should go back to ascii and HTML encoding. Makes sense to me.

**Comment by HLeithner** [comment]
forget the "idiots", I already argument like other people in this tracker and that makes it sick. People are doing anyway what they want to do so I should not take this personally and look at my health.

**Comment by brianteeman** [comment]
I know many of the unicode consortium, spoken at their conference twice, nice people. However, unfortunately, much of their work is not accessible at all or more accurately is misused and is therefore not accessible.

**Comment by brianteeman** [comment]
No one was forced to merge this

**Comment by N6REJ** [comment]
is it possible to add the &hellip; to tiny as I'm darn sure never going to remember that and I use ... almost hourly

**Comment by brianteeman** [comment]
What does that have to do with changing language strings?

**Pull request merged**